### DISCUSSION TITLE
How do I let a page slide in and out from the left on iOS instead of right?

### DISCUSSION THREAD

**Discussion by bathientran** [react-navigation/react-navigation#11472] [Q&A]
I'm trying to set up a screen on the left and right of the main screen and each of them should "slide-in" and "slide-out" from their respective sides. How can I achieve this on iOS?

**Comment by sanctas** [comment]
You cannot. I have tried so many things to achieve similar behavior. It is simply not supported. It would require a rework of the entire structure of the stack navigator, so much so you'd be better off just implementing this yourself or using a different nav package altogether (although based on your requirements none will likely work). 

It's a shame something like this doesn't exist.
